packages/Webkul/Product/src/Type/Configurable.php

```diff
@@ -198,6 +199,7 @@ public function update(array $data, $id, $attribute = 'id')
 
                     $variantData['locale'] = $data['locale'];
 
+                    $variantData['tax_category_id']=$data['tax_category_id'];
```

Comment:
Alway use space before and after with binary operator. And also add new line after this statement